### ISSUE TITLE
Bug: space-after-colon warning when trailing-semicolon is set to false.

### ISSUE THREAD

**Issue by whaaaley** [sasstools/sass-lint#744]
I'm using `sass-lint` through the Atom plugin `linter-sass-lint`. In the screenshot you can see Atom's SCSS syntax highlighter is also bugged in the same scenario, however appears to be valid SCSS syntax. The dot and underline shows where the warning is coming from.

![img](http://f.cl.ly/items/430H3A1M3Q2W2o1j1d1m/Screen%20Shot%202016-06-19%20at%2011.49.00%20AM.png)

---

When the trailing semicolon are set to be removed the next block with a colon throws this warning.

![img](http://cl.ly/0s2E1U1r092Z/Screen%20Shot%202016-06-19%20at%2011.58.42%20AM.png)

---

Here's the snippet from my `.sass-lint.yml`

``` yaml
  trailing-semicolon:
    - 1
    - include: false
```
